src/components/MainMenu.vue

```diff
@@ -28,11 +28,17 @@ export default {
         name: 'B Tree',
         imageUrl: 'https://cdn.cienradios.com/wp-content/uploads/sites/4/2020/02/pablito-lescano.gif',
         code: 'b-tree',
-      }, {
+      },
+      {
         name: 'Min Heap',
         imageUrl: 'https://cdn.cienradios.com/wp-content/uploads/sites/4/2020/02/pablito-lescano.gif',
         code: 'min-heap',
       },
+      {
+        name: 'Max Heap',
+        imageUrl: 'https://cdn.cienradios.com/wp-content/uploads/sites/4/2020/02/pablito-lescano.gif',
```

Comment:
In #41 I added real images related with the structure. Try to find one appropiate so we can have a nice main menu :)

Reply:
Ok, gonna look for some nice max heap img